functional_tests/identify.test.ts

```diff
@@ -7,6 +7,7 @@ import { PostHog } from '../src/posthog-core'
 import { logger } from '../src/utils/logger'
 import { uuidv7 } from '../src/uuidv7'
 import { getRequests } from './mock-server'
+import { IDENTIFY_EVENT, SET_EVENT } from '../src/events'
```

Comment:
I wonder if yall prefer it this way or something like `EVENTS.IDENTIFY` instead

Reply:
the problem with TS enums are mostly when they are used as numbers. as long as we keep enums as string values only they're fine

Reply:
> repeated hardcoded strings\numbers though

In a way `const RAGECLICK_EVENT = '$rageclick'` is also repeating it, which does not spark joy

---

Given the comments and my great dislike of enums, I'll vote for keeping this as is. We can easily change this in the future, I'm actually pretty sure a future contributor will change that exact thing :)